src/plugins/syscalls/linux.h

```diff
@@ -210,224 +166,742 @@ SYSCALL(openat, LONG,
     "flags",    "", DIR_IN, ULONG,
     "mode",     "", DIR_IN, ULONG,
 );
-
-// TODO: fill in missing ret & argument info
-SYSCALL(stat, VOID);
-SYSCALL(fstat, VOID);
-SYSCALL(lstat, VOID);
-SYSCALL(poll, VOID);
-SYSCALL(lseek, VOID);
-SYSCALL(mmap, VOID);
-SYSCALL(mprotect, VOID);
-SYSCALL(munmap, VOID);
-SYSCALL(brk, VOID);
-SYSCALL(rt_sigaction, VOID);
-SYSCALL(rt_sigprocmask, VOID);
-SYSCALL(rt_sigreturn, VOID);
-SYSCALL(ioctl, VOID);
-SYSCALL(pread64, VOID);
-SYSCALL(pwrite64, VOID);
-SYSCALL(readv, VOID);
-SYSCALL(writev, VOID);
-SYSCALL(access, VOID);
-SYSCALL(pipe, VOID);
-SYSCALL(select, VOID);
+SYSCALL(stat, LONG,
+    "pathname", "", DIR_IN, PCHAR,
+    "statbuf",  "", DIR_IN, PVOID,
+);
```

Comment:
Almost all of them are taken from this list https://chromium.googlesource.com/chromiumos/docs/+/master/constants/syscalls.md#x86_64-64_bit

It was easier to move them by hand, because it was necessary not only to transfer all the syscalls, but also to check their usefulness